examples/Db2/variables.tf

```diff
@@ -1,8 +1,14 @@
-variable "cluster_name_or_id" {
+variable "cluster_id" {
   default     = ""
   description = "Enter your cluster id or name to install the Cloud Pak. Leave blank to provision a new Openshift cluster."
 }
 
+//variable "cluster_config_path"  = ".kube/config" {
```

Comment:
Eric just dropped a fix to make this a global variable again.  Make sure this doesn't undo his fix.  Not sure why this is commented out.